source/ChromeDevTools/CommandResponseWrapper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace MasterDevs.ChromeDevTools
+{
+    public interface ICommandResponseWrapper<T>
```

Comment:
@svatal any reason not to merge the ICommandResponse and ICommandResponseWrapper?

Reply:
I have been afraid how deserialization would cope with a class that have something more than a basic properties. Did not tested it though.